APIs.md

```diff
@@ -20,6 +20,15 @@ model, to be created and edited using simple HTTP messages.
 The OpenSearch catalogue allows you to search through Copernicus data using a standardized web service. Search can be performed based on multiple attributes such as collection, spatial extent, time range, metadata. Orders can be based
 on the data collection, area, time range or any other selection criteria available in the search APIs
 
+### Catalog APIs news
+This section provides detailed information about upcoming changes and the changelog of the implemented updates. The releases include all APIs interfaces. To avoid disruption to your scripts or apps, we recommend reviewing the upcoming changes and the latest release notes described below:
+
+#### 1. Upcoming Changes
+We've put together a list of potential updates related to OpenSearch and the OData API as of September 2023. ou can find a detailed comparison of these changes [here](APIs/UpcomingChanges.qmd).
```

Comment:
'ou' typo?

Reply:
Fixed in: https://github.com/eu-cdse/documentation/pull/289/commits/e8003b3bf806acfa1158985ef7e9012bcdac4ca8